cla-backend-go/v2/cla_groups/service.go

```diff
@@ -705,10 +705,25 @@ func getS3Url(claGroupID string, docs []v1Models.ProjectDocument) string {
 
 // ListClaGroupsForFoundationOrProject returns the CLA Group list for the specified foundation ID
 func (s *service) ListClaGroupsForFoundationOrProject(foundationSFID string) (*models.ClaGroupList, error) {
+	var v1ClaGroups = new(v1Models.Projects)
 	out := &models.ClaGroupList{List: make([]*models.ClaGroup, 0)}
-	v1ClaGroups, err := s.v1ProjectService.GetClaGroupsByFoundationSFID(foundationSFID, DontLoadDetails)
-	if err != nil {
-		return nil, err
+	var err error
+
+	projectDetails, projDetailsErr := v2ProjectService.GetClient().GetProject(foundationSFID)
+	if projDetailsErr != nil {
+		log.Warnf("unable to lookup foundation/project SFID: %s - error: %+v", foundationSFID, projDetailsErr)
+	}
+	if projectDetails.ProjectType == "Project" {
+		v1ClaGroupsByProject, prjerr := s.v1ProjectService.GetClaGroupByProjectSFID(foundationSFID, DontLoadDetails)
+		if prjerr != nil {
+			return nil, prjerr
+		}
+		v1ClaGroups.Projects = append(v1ClaGroups.Projects, *v1ClaGroupsByProject)
+	} else {
```

Comment:
Could you change the else to check if project is Foundation(Now Project Group) . Something like
```else if projectDetails.ProjectType == "Project Group" {}``` and handle the other condition (project invalid..not either Project or Project Group Type)